libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -7,20 +7,21 @@ import {Reducer, useReducer} from 'react'
 import {MdClose, MdMenu} from 'react-icons/md'
 import {navigationLinkToUrl} from '../link-to-url'
 
-export const NavbarWrapper = styled('nav')<{isMenuOpen: boolean}>`
+export const NavbarWrapper = styled('nav')`
```

Comment:
Fixes not being able to set a background-color without breaking the open menu background-color